apps/frontend/pages/404.tsx

```diff
@@ -0,0 +1,33 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.wrapper}>
+
+      <h1 className={styles.mainH}><b>404.</b></h1>
+
+      <a className={styles.properText}>Może szukałeś czegoś innego? Sprawdź to:</a>
+
+      <div className={styles.divlist}>
+
+      <ul className={styles.listmenu}>
+        <li>
+          <a className={styles.mainref} href="/">Strona Główna</a>
```

Comment:
each `a` should have the same `className`
what's the point of creating a separate one for each?